packages/shared/src/hooks/useFeedSettings.ts

```diff
@@ -53,14 +54,29 @@ export default function useFeedSettings({
   );
 
   const { tagsCategories, feedSettings, advancedSettings } = feedQuery;
+  const checkEnabledSettings = useCallback(
+    (value: number) => {
```

Comment:
Value is the advancedSettingsId?
I would maybe call it that or `id` just so it's clear.

Reply:
Right, I was reserving the `id` on the destructure part, but we can do that.